src/realm/sync/noinst/client_reset.hpp

```diff
@@ -58,7 +58,8 @@ struct ClientResetFailed : public std::runtime_error {
 //
 // The result is that src group is unchanged and the dst group is equal to src
 // when this function returns.
-void transfer_group(const Transaction& tr_src, Transaction& tr_dst, util::Logger& logger);
+void transfer_group(const Transaction& tr_src, Transaction& tr_dst, util::Logger& logger,
```

Comment:
Maybe instead of a default arg, just force the caller to specify whether we allow adding to the schema or not...

Reply:
Done.